src/TEC/Import/Post_Handler.php

```diff
@@ -131,9 +131,24 @@ public function maybe_create_post( bool $continue_import, array $data, int $impo
 			! isset( $data['posttype'] )
 			|| ! in_array( $data['posttype'], $this->get_supported_post_types( false ), true )
 		) {
-			$this->add_to_log( "Not supported post type. Skipping." );
+			$msg = "Not supported post type. ";
 
-			return false;
+			/**
+			 * A filter to allow forcing the import even if post type is not supported or not set.
+			 *
+			 * @since 1.1.0
+			 *
+			 * @param bool $continue Whether the import should be forced to continue. Default: false.
+			 */
+			$continue = apply_filters( 'tec_labs_wpai_is_post_type_set', false );
+
+			$msg .= $continue ? '`tec_labs_wpai_is_post_type_set` override in place, post will be imported.' : 'Skipping.' ;
+			$this->add_to_log( $msg );
+
+			// Bail, if there is no override for the non-supported post type.
+			if ( ! $continue ) {
+				return false;
+			}
```

Comment:
```suggestion
		return $continue;
```

You can return early with the `$continue` varialbe itself. Continueing with a posttype not supported or empty requires more changes :D

Reply:
I guess with the new filter we can disregard this?

Reply:
Yes!